dev.env.example

```diff
@@ -0,0 +1,6 @@
+DBT_TEST_ATHENA_S3_STAGING_DIR=
```

Comment:
I have a intelij plugin which recognizes `*.env`, so for me `example.dev.env` would be better.